README.md

```diff
@@ -23,9 +23,21 @@ Commit your code regularly and meaningfully. This helps both you (in case you ev
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit the `ANSWERS.md` file to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 - [ ] Why would you use class component over function components (removing hooks from the question)?
+
+1. Class components are allowed to fetch data plus handle changes unlike our easy to write fuctional components.
+
```

Comment:
Our easy to write functional components are also allowed to fetch data and handle changes. What specifically do you think will make you choose class components over functional components?